main.js

```diff
@@ -26,12 +26,23 @@ const isBrowser = (typeof self !== 'undefined');
  * @param {object} options.invocationSigner - The invoker's key for signing.
  * @param {string} options.capabilityAction - The action(s) the capability
  *   can perform.
+ * @param {string|Date|number} [options.created = Date.now()] - created is a
+ *  psuedo-header used in the http signature.
+ * @ param {string|Date|number} [options.expires = null] - expires is a
+ *  psuedo-header used to ensure the header expires.
```

Comment:
```suggestion
 *   psuedo-header used in the http signature.
 * @param {string|Date|number} [options.expires] - expires is a
 *   psuedo-header used to ensure the header signature expires.
```

Not sure if our docs pattern is to be explicit with `[options.expires = undefined]` or if `[options.expires]` will do (I'm suggesting we use `undefined`, not `null` for its default).